cmp/grid/GridModel.js

```diff
@@ -352,14 +346,15 @@ export class GridModel {
     noteAgColumnStateChanged(agColState) {
         const colChanges = agColState.map(({colId, width, hide}) => {
             const col = this.findColumn(this.columns, colId);
+            if (!col) return null;
             return {
                 colId,
                 hidden: hide,
                 width: col.flex ? undefined : width
             };
         });
 
-        this.applyColumnChanges(colChanges);
+        this.applyColumnChanges(colChanges.filter(Boolean));
```

Comment:
I find this filter statement an awkward way to remove nulls.  Doesn't lodash have a better way to remove nulls?

Reply:
`without(colChanges, null)`?